var/spack/repos/builtin/packages/py-callmonitor/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyCallmonitor(PythonPackage):
+    """A Simple Tool to Monitor and Log Function Calls"""
+
+    homepage = "https://github.com/JBlaschke/call-monitor"
+    pypi = "callmonitor/callmonitor-0.3.7.tar.gz"
+
+    maintainers("DaxLynch")
+
+    version("0.3.7", sha256="11bacfe5940c3f6aff223e8e761b033d540542b4d738f7fef38cd923b3be0cbc")
+
+    depends_on("py-setuptools", type="build")
```

Comment:
`py-numpy` is missing as dependency (see `install_requires` in `setup.py`

Reply:
Done!